### PR TITLE
add stale

### PR DESCRIPTION
**Pull request by casz**
### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!

- [x] Please describe what you did

- [ ] Link to relevant GitHub issues or pull requests

- [ ] Link to relevant [Jenkins JIRA issues](https://issues.jenkins-ci.org)

- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### Description

Let stale manage our inactive issues and pull requests.
Do the current settings seem unreasonable?

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+
+# Label to use when marking as stale
+staleLabel: wontfix
```
Comment (timja) [inline .github/stale.yml]: can't we just go stale? wontfix may not be right

Comment (casz) [reply .github/stale.yml]: Done, that was just the copy-pasted example. :sweat_smile:

**Comment by ewelinawilkosz** [comment]
I'll have a look today or tomorrow!

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
```
Comment (casz) [inline .github/stale.yml]: `enhancement` might be another good exemption.

Comment (ewelinawilkosz) [reply .github/stale.yml]: I am happy to merge that one, and see where it takes us :) do you want to add enhancement though?

Comment (timja) [reply .github/stale.yml]: sounds worth adding

**Comment by casz** [comment]
Let's see what this does 😓

**Pull request merged**